### ISSUE TITLE
Thésaurus : ordre d'apparition des données par défaut : alphabétique ou de cote

### ISSUE THREAD

**Issue by Marion-L1** [Francoralite/francoralite#458]
**Mode**
- [x] Consultation
- [ ] Saisie

**Type**

- place des informations

**Problème**
Dans les différentes pages de l'onglet thésaurus (enquêteurs, informateurs, langues,....) les données apparaissent dans un ordre aléatoire.

**Descriptif**
Faire en sorte que les données principales de toutes les pages (sauf des catalogues) soient triées par ordre alphabétique, par défaut.
Faire en sorte que les données des Catalogues (Coirault, Laforte, Aarne-Thompson et Delarue-Thenèse) soient triées par cote, par défaut.

**Comment by lluc** [comment]
@Marion-L1  Les langues sont affichées dans l'ordre alphabétique de leur nom. Faut-il modifier ce classement ?

**Comment by Marion-L1** [comment]
> @Marion-L1 Les langues sont affichées dans l'ordre alphabétique de leur nom. Faut-il modifier ce classement ?

Non, c'est très bien comme ça, tu peux le laisser dans cet ordre de classement. Merci.

**Comment by Marion-L1** [comment]
Précision : pour le catalogue des timbres ça sera une classification par ordre alphabétique, contrairement aux autres catalogues.

**Comment by bpradines** [comment]
@Marion-L1 @lluc Classement des thésaurus par cote : 
- Les thésaurus Aarne-Thompson et Delarue-Thenèse n'ont pas de cote ;
- Les thésaurus Coirault et Laforte en ont, mais si on cherche un titre en particulier, ce qui semblait être l'idée du classement alphabétique, n'est-il pas préférable plutôt de classer par titre ?

**Comment by Marion-L1** [comment]
> @Marion-L1 @lluc Classement des thésaurus par cote :
> 
>     * Les thésaurus Aarne-Thompson et Delarue-Thenèse n'ont pas de cote ;
> 
>     * Les thésaurus Coirault et Laforte en ont, mais si on cherche un titre en particulier, ce qui semblait être l'idée du classement alphabétique, n'est-il pas préférable plutôt de classer par titre ?

* Thésaurus Delarue-Thenèze : existence de cotes de T.001 à T.999
*  Thésaurus Aarne-Thompson : existence de cotes de AT 001 à AT 499

Dans toutes ces classifications, les thématiques sont plus importantes que les titres (qui ont d'ailleurs été attribués dans un second temps). Donc le classement par ordre de cotes est préférable.

**Comment by bpradines** [comment]
@Marion-L1 Pour l'ordre des thésaurus Coirault et Laforte : un classement par cote est possible, mais dans une future version.

**Comment by Marion-L1** [comment]
> @Marion-L1 actuellement on ne dispose d'aucune donnée pour réaliser les thésaurus Delarue-Thenèze et Aarne-Thompson. Il faut nous fournir les données !!!

Nous sommes en cours de travail actuellement pour vous fournir le plus rapidement possible les thésaurus Coirault, Laforte et Delarue-Thenèze sous des versions facilement intégrables.
